config/owasp/suppressions.xml

```diff
@@ -23,5 +23,6 @@
     <cve>CVE-2020-13935</cve>
     <cve>CVE-2020-8022</cve>
     <cve>CVE-2017-18640</cve>
+    <cve>CVE-2020-5421</cve>
```

Comment:
Please add a note mentioning the reason for suppressing the CVE

Reply:
The above CVE is coming for spring modules with version 5.2.4 i.e spring - aop, spring -aspects, spring, etc. These are transactive dependencies of spring boot version 2.2.5.  Spring boot version upgrade needs to happen to resolve these CVEs.

Reply:
Makes sense, thanks @amitchidrewar1301